share/cmake/utils/CheckSupportGL.cmake

```diff
@@ -35,6 +36,35 @@ if((OCIO_BUILD_TESTS AND OCIO_BUILD_GPU_TESTS) OR OCIO_BUILD_APPS)
                 set(GLEW_LIBRARIES GLEW::GLEW)
             endif()
         endif()
+
+        # Cmake has a bug with FindGLEW. It fails to defined GLEW_INCLUDE_DIRS in some situation.
+        # See https://gitlab.kitware.com/cmake/cmake/-/issues/19662
+        # See a closed duplicate of 19662: https://gitlab.kitware.com/cmake/cmake/-/issues/20699
+        # Ported from vcpkg Glew package - https://github.com/microsoft/vcpkg/blob/master/ports/glew/vcpkg-cmake-wrapper.cmake
+        # The following code make sure that GLEW_INCLUDE_DIRS is set correctly.
+        if(GLEW_FOUND AND TARGET GLEW::GLEW AND NOT DEFINED GLEW_INCLUDE_DIRS)
+            get_target_property(GLEW_INCLUDE_DIRS GLEW::GLEW INTERFACE_INCLUDE_DIRECTORIES)
+            set(GLEW_INCLUDE_DIR ${GLEW_INCLUDE_DIRS})
+            get_target_property(_GLEW_DEFS GLEW::GLEW INTERFACE_COMPILE_DEFINITIONS)
+            if("${_GLEW_DEFS}" MATCHES "GLEW_STATIC")
+                get_target_property(GLEW_LIBRARY_DEBUG GLEW::GLEW IMPORTED_LOCATION_DEBUG)
+                get_target_property(GLEW_LIBRARY_RELEASE GLEW::GLEW IMPORTED_LOCATION_RELEASE)
+            else()
+                get_target_property(GLEW_LIBRARY_DEBUG GLEW::GLEW IMPORTED_IMPLIB_DEBUG)
+                get_target_property(GLEW_LIBRARY_RELEASE GLEW::GLEW IMPORTED_IMPLIB_RELEASE)
+            endif()
+            get_target_property(_GLEW_LINK_INTERFACE GLEW::GLEW IMPORTED_LINK_INTERFACE_LIBRARIES_RELEASE) # same for debug and release
+            list(APPEND GLEW_LIBRARIES ${_GLEW_LINK_INTERFACE})
+            list(APPEND GLEW_LIBRARY ${_GLEW_LINK_INTERFACE})
```

Comment:
This looks like undefined in Linux which breaks CI, should this line be removed as we don't seem to use `GLEW_LIBRARY` elsewhere? Is this variable only created on Windows? CMake [documentation](https://cmake.org/cmake/help/latest/module/FindGLEW.html) doesn't mention it.

Reply:
You're right. I don't see it used anywhere. I've seen some mentioned of GLEW_LIBRARY, but they are all from before 2019/2018 with an older version of cmake.

I'll remove it as it is not needed on Windows either.